cdci_data_analysis/flask_app/app.py

```diff
@@ -90,8 +84,24 @@ def run_api_parameters():
 
 @app.route("/api/instr-list")
 def run_api_instr_list():
-    instrument_list = InstrumentQueryBackEnd.get_user_specific_instrument_list(app)
-    return instrument_list
+    logger.warning('\nThe endpoint \'/api/instr-list\' is deprecated and you will be automatically redirected to the '
+                   '\'/instr-list\' endpoint. Please use this one in the future.\n')
+
+    bind_host = app.config['conf'].bind_host
+    bind_port = app.config['conf'].bind_port
+
+    parsed_product_url = urlparse(app.config['conf'].products_url)
+
+    if (parsed_product_url.hostname is None and parsed_product_url.port is None) or \
+            (parsed_product_url.hostname == bind_host and parsed_product_url.port == bind_port):
+        redirection_url = url_for('instr_list')
+    else:
+        redirection_url = os.path.join(app.config['conf'].products_url, 'dispatch-data/instr-list')
+
+    if request.args:
+        redirection_url += f'?{urlencode(request.args)}'
```

Comment:
do we only have `get` here?

Reply:
yes

Reply:
where is it restricted?

Reply:
are we talking about the method for this endpoint?

Reply:
yes

Reply:
default to GET